libraries/AP_Math/tests/test_perpendicular.cpp

```diff
@@ -0,0 +1,38 @@
+#include <AP_gtest.h>
+
+#include "/home/pbarker/rc/ardupilot/ArduCopter/avoidance_handler.h"
+
+#define SQRT_2 1.4142135623730951f
```

Comment:
I am now complaining about the include line :)
On Jun 9, 2016 7:59 PM, "Francisco Ferreira" <notifications@github.com>
wrote:

In libraries/AP_Math/tests/test_perpendicular.cpp
<https://github.com/ArduPilot/ardupilot/pull/4278#discussion_r66556232>:

> @@ -0,0 +1,38 @@
> +#include <AP_gtest.h>
> +
> +#include "/home/pbarker/rc/ardupilot/ArduCopter/avoidance_handler.h"
> +
> +#define SQRT_2 1.4142135623730951f

And me thinking you were going to complain about the include line 😄

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/ArduPilot/ardupilot/pull/4278/files/326d9536a7c61c80d3a23109103d8d05f0072053#r66556232>,
or mute the thread
<https://github.com/notifications/unsubscribe/AEj7G_DBX5__AfpoBn_nSbcLI4JE6BJWks5qKNL8gaJpZM4Iul1O>
.